tests/test_storage_locations_handler.py

```diff
@@ -0,0 +1,45 @@
+"""
+Copyright 2023 NOAA
+All rights reserved.
+
+Unit tests for storage_locations.py
+
+"""
+import pytest 
+from storage_locations import StorageLocationRequest
+
+
+def test_storage_location_input_dict():
+    request_dict = {
+        'name': 'storage_locations',
+        'method': 'PUT',
+        'body': {
+            'name': 's3_example_bucket',
+            'platform': 'aws_s3', 
+            'bucket_name': 'noaa-example-score-db-bucket',
+            'key': 'noaa-exmample-score-db-bucket/reanalysis',
```

Comment:
should the key really have a bucket_name in it? Seems like it will not in a real world?